### DISCUSSION TITLE
Cant open live server while on Codespaces

### DISCUSSION THREAD

**Discussion by epeltz33** [community/community#77579] [Codespaces]
### Select Topic Area

Bug

### Body

Hello all, 
So I am using a codespace and I am trying tgo open a file in my repo with the extension Live Server. I keep getting an error saying the connection is not private. So I made sure that the port is using http and its visibility is public, but that didnt help. I also tried opening the file with the preview option in codespaces and I got the same error. I looked in the console and I saw this error "Refused to set unsafe header "User-Agent". I will attach a screenshot of the error but if anyone could help me fix this bug I would appreciate it ! 
![Screenshot 2023-12-01 at 2 15 25 PM](https://github.com/community/community/assets/86162368/0d3b0ea0-f097-4a59-a93d-480ef744fa89)

**Comment by shyakachaste** [comment]
hello

**Comment by shyakachaste** [comment]
To resolve the first issue, ensure your Live Server is configured for HTTP instead of HTTPS, or verify and fix any SSL certificate problems if HTTPS is required. For the second issue, check your JavaScript code and third-party scripts to eliminate any attempts to set the 'User-Agent' header, as browsers restrict the direct setting of certain headers for security reasons. After addressing these points, retry running your Live Server to see if the problems persist.

**Reply by epeltz33** [reply]
Awesome, it was a third party script that was causing the issue. Thanks for the help!

**Reply by shyakachaste** [reply]
you are welcome

**Reply by shyakachaste** [reply]
@epeltz33 remember to mark as solved, bcz i continue to see that it was not solved
